src/js/components/award/shared/InfoTooltipContent.jsx

```diff
@@ -1249,3 +1249,22 @@ export const CFDASectionInfo = (
         </div>
     </div>
 );
+
+export const recipientLoanFaceValueInfo = (
+    <div className="award-summary-tooltip">
```

Comment:
You're doing the right thing here by reusing all this stuff, but can we adjust this className to reflect where it's being used and put it somewhere in the recipient directory?